vrk-kpa/suomidigi
# SUOPA-686 Group frontpage

To test: 
- Check this PR first https://github.com/vrk-kpa/suomidigi/pull/122
- `make fresh`
- Go to https://suomidigi.docker.sh/node/add/community
   - Fill in all the fields and save the node
- Go to https://suomidigi.docker.sh/admin/group and click on **Add group**
   - Fill in the fields. Try to add new paragraphs **Contact information** and **List of links**
   - For community reference, search for the community which was added in last step
   - When all is done, click save
- From the group page, click on **Members**
   - Click on **Add member**, add a member to the group and repeat this multiple times
- From the group page, click on **Nodes**
   - Click on **Add existing content**, add few **articles** to the group
   - Click on **Add new content**, add a couple of **events** to the group
- After all of these steps, the Group front page should be filled with content. Head to the group page and test it in mobile and desktop widths. 
Compare it to attachment pictures found in here: https://vrk-ejira.eden.csc.fi/browse/SUOPA-686
